todo/urls.py

```diff
@@ -0,0 +1,31 @@
+from django.urls import path
+
+from .views import (
+    TaskListView,
+    TaskCreateView,
+    TaskUpdateView,
+    TaskDeleteView,
+    TagListView,
+    TagCreateView,
+    TagUpdateView,
+    TagDeleteView,
+    TaskChangeStatusView,
+
+)
+
+urlpatterns = [
+    path("", TaskListView.as_view(), name="task-list"),
+    path("task/create", TaskCreateView.as_view(), name="task-create"),
+    path("task/<int:pk>/update", TaskUpdateView.as_view(), name="task-update"),
+    path("task/<int:pk>/delete", TaskDeleteView.as_view(), name="task-delete"),
+
+    path("tags/", TagListView.as_view(), name="tag-list"),
+    path("tags/create", TagCreateView.as_view(), name="tag-create"),
+    path("tags/<int:pk>/update", TagUpdateView.as_view(), name="tag-update"),
+    path("tags/<int:pk>/delete", TagDeleteView.as_view(), name="tag-delete"),
+
+    path('chng_st/<int:pk>', TaskChangeStatusView.as_view(), name='task_change_status'),
```

Comment:
You should use double quotes instead of single quotes here and potentially in other cases to be consistent with project structure. For the same reason you should change `task_change_status` to `task-change-status`. Besides that, endpoint naming shortening is not right, use more descriptive naming or move the functionality of this endpoint to the `"task/<int:pk>/update"` endpoint.